CMakeLists.txt

```diff
@@ -53,6 +96,8 @@ if(DEFINED VCPKG_ROOT)
   if(NOT DEFINED CMAKE_TOOLCHAIN_FILE)
     set(CMAKE_TOOLCHAIN_FILE "${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake" CACHE STRING "")
   endif()
+else()
+  message(STATUS "Using dependencies from system without vcpkg")
```

Comment:
Remove? This is not relevant for Linux.

Reply:
It is relevant for Linux. vcpkg can be used on Linux and this is helpful to show whether it is being used or not.

Reply:
OK.